lib/layout.js

```diff
@@ -215,8 +215,10 @@ El.calcSize = (el) => (
 
 El.calcScrollSize = (el) => (
   el === window
-    ? { w: el.scrollX, h: el.scrollY }
-    : { w: el.scrollLeft, h: el.scrollTop }
+   ? {
+    w: el.scrollX || el.pageXOffset || document.documentElement.scrollLeft,
+    h: el.scrollY || el.pageYOffset || document.documentElement.scrollTop
```

Comment:
for more browser coverage, [see here](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY?redirectlocale=en-US&redirectslug=DOM%2Fwindow.scrollY)